src/App.jsx

```diff
@@ -14,70 +15,68 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selected, setSelected] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const onClick = (product) => {
+    setSelected(selected === product
+      ? 'No goods selected'
+      : product);
+  };
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  // const isSelected = selected === good;
+  const isSelected = good => selected === good;
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      {selected !== 'No goods selected'
```

Comment:
```suggestion
      {selected !== NONE
```
Consider using a constant value here ```const NONE = 'No goods selected'``` instead of a magic string